tests/math/test_matrix_manipulation.py

```diff
@@ -23,6 +22,22 @@
 import pennylane as qml
 from pennylane import numpy as pnp
 
+tf = pytest.importorskip("tensorflow", minversion="2.1")
+torch = pytest.importorskip("torch")
+jax = pytest.importorskip("jax")
+jnp = pytest.importorskip("jax.numpy")
```

Comment:
remove those

Reply:
removed these, just a heads up, this was to help with black formatting which is failing bc the imports were outside the imports portion of the script

Reply:
`black` _shouldnt_ be a problem, make sure to be on the latest version by `pip install --upgrade black`

Reply:
And pylint has an exception for these imports in the `tests/.pylintrc` file